<launch>

  <node name="sherd_states" pkg="robot_arm" type="sherd_states.py" output="screen">
    <rosparam command="load" file="$(find robot_arm)/config/mat_layout.yaml"/>
  </node>

    <node name="read_scale_server" pkg="robot_arm" type="read_scale_server.py" output="screen" respawn="true"/>

    <node name="take_photo_server" pkg="robot_arm" type="take_photo_server.py" output="screen" respawn="true"/>

</launch> 
